### ISSUE TITLE
Jetty 9.4 sets X-Frame-Options to SAMEORIGIN

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#2639]
Jetty 9.4 with https://github.com/eclipse/jetty.project/pull/1613 sets `X-Frame-Options: SAMEORIGIN` by default.

This prevents apps-embedding in xframes on other sites, and thus should be avoided because it is largely used.

=> Configure Jetty's `HeaderFilter` at least on SP but probably also on other apps (to be checked), from `18.06` on.

**Comment by pmauduit** [comment]
I think it is more of spring-security than jetty. From what I undertood, nothing in the PR pointed above indicates it's activated by default, and I cannot reproduce with a fresh jetty:9.4 docker image:

```
 % curl localhost:8080/sample/ -i
HTTP/1.1 200 OK
Content-Type: text/html;charset=utf-8
Set-Cookie: JSESSIONID=node012bjccyirq4h1dalcvi48v5p61.node0;Path=/sample
Expires: Thu, 01 Jan 1970 00:00:00 GMT
Content-Length: 52
Server: Jetty(9.4.18.v20190429)

<html>
<body>
<h2>Hello World!</h2>
</body>
</html>

```

**Comment by fvanderbiest** [comment]
My bad. You're right.
And I see that @RemiDesgrange already has a fix with #2641 !

**Comment by RemiDesgrange** [comment]
and #2642 and #2643 :-) for master ans 19.04 branch
